test/commandLine.test.js

```diff
@@ -34,30 +35,30 @@ describe('CommandLineManager', function () {
       argv = argv.concat(['--proxy-host', 'host', '--proxy-port', '9687']);
       CommandLineManager.processArgs(argv);
       console.log.restore();
-      expect(RdGlobalConfig.proxy.host).to.eql('host');
+      expect(RdGlobalConfig.proxy.host).to.eql(proxy_host_actual_value);
       expect(RdGlobalConfig.proxy.port).to.eql(9687);
     });
 
     it('remove any protocol part from proxy-host', function () {
       sinon.stub(console, 'log');
-      argv = argv.concat(['--proxy-host', 'http://host']);
+      argv = argv.concat(['--proxy-host', 'host']);
       CommandLineManager.processArgs(argv);
-      expect(RdGlobalConfig.proxy.host).to.eql('host');
+      expect(RdGlobalConfig.proxy.host).to.eql(proxy_host_actual_value);
     });
 
     it('remove any prefix slashes from proxy-host', function () {
```

Comment:
These were added for their purpose. Please check if these are relevant anymore since the changes doesn't align with the description anymore.

Reply:
Protocol part is required for http-proxy-agent/http-proxy-agent to work.

Reply:
can be resolved. The specs are removed now.